src/app/characters/character-list/character-list.component.ts

```diff
@@ -1,7 +1,9 @@
 import { Component, OnInit, ViewChild } from '@angular/core';
 import { MatDrawer } from '@angular/material/sidenav';
-import { EMPTY, Observable, of, Subject } from 'rxjs';
-import { catchError, debounceTime, distinctUntilChanged, filter, switchMap } from 'rxjs/operators';
+import { MatSnackBar, MatSnackBarRef, TextOnlySnackBar } from '@angular/material/snack-bar';
+import { EMPTY, Observable, Subject } from 'rxjs';
+import { catchError, debounceTime, distinctUntilChanged, filter, map, switchMap } from 'rxjs/operators';
+import { environment } from 'src/environments/environment';
```

Comment:
Could you please change that to use relative module import order and align it with the other imports below? (These are all part of the ESLint configuration that I plan to integrate soon)

Reply:
Done.

Reply:
I really like to use lint rules and that was a rule I did not know! I look forward to seeing the new ESLint configuration.